tests/functional/nonpipeline-stream-vadd/vadd.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2024 RapidStream Design Automation, Inc. and contributors.
```

Comment:
What does it mean (nonpipeline)? If it means the loops, Vitis will automatically insert pipeline for loops sometimes.